daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/processors/PackedBinaryTraits.scala

```diff
@@ -51,7 +52,14 @@ abstract class PackedBinaryDecimalBaseParser(
 
   def parse(start: PState): Unit = {
     val nBits = getBitLength(start)
-    if (nBits == 0) return // zero length is used for outputValueCalc often.
+    if (nBits == 0) {
+      PE(
+        start,
```

Comment:
This does need test coverage. It may be possible to remove this check entirely. It may not be reachable in that something else is checking for the zero case before this is called?

Reply:
It is actually reachable. I've added the test. I don't think we ought to remove this check as we don't check for nBits == 0 anywhere else